libs/designsystem/progress-circle/src/progress-circle-ring.component.scss

```
@@ -9,13 +9,18 @@ svg {
 }
 
 .progress {
-  transition-property: stroke-dashoffset, stroke;
+  transition-property: stroke-dasharray, stroke;
   transition-duration: utils.get-transition-duration('extra-long');
   transition-timing-function: utils.get-transition-easing('enter-exit');
   transform-origin: 50% 50%;
   stroke: var(--kirby-progress-circle-stroke-color, #{utils.get-color('success')});
 }
 
+// Avoid flash of full progress circle if attribute binding in template has not yet been evaluated.
+.progress:not([stroke-dasharray]) {
+  stroke-dasharray: 0 999;
+}
+
```

Consider grouping with rest of `.progress` styles:
```suggestion
  stroke: var(--kirby-progress-circle-stroke-color, #{utils.get-color('success')});
  // Avoid flash of full progress circle if attribute binding in template has not yet been evaluated.
  &:not([stroke-dasharray]) {
    stroke-dasharray: 0 999;
  }
}
```